charts/tempo-distributed/templates/distributor/deployment-distributor.yaml

```
@@ -52,6 +52,8 @@ spec:
       hostAliases:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+        {{- toYaml .Values.distributor.initContainers | nindent 8 }}
```

I recommend this syntax, otherwise it might be lead to diffs in GitOps tools.
```suggestion
      {{- with .Values.distributor.initContainers }}
      initContainers:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```